Sources/Defaults/Utilities.swift

```diff
@@ -327,21 +338,9 @@ final class TaskQueue {
 	Queue a new asynchronous task.
 	*/
 	func async(_ task: @escaping AsyncTask) {
+		lock.lock()
```

Comment:
It's generally better to have a `.with` type method to make sure the locked resource is always unlocked. Like: https://developer.apple.com/documentation/os/osallocatedunfairlock/4061628-withlock